shared/markdown/parser.pegjs

```diff
@@ -167,26 +166,26 @@ NativeEmoji
 LinkChar
  = !(SpecialChar+ (LineTerminatorSequence)) char:NonBlank { return char }
 
-Link 
+Link
   = url:( [([]* ("http"i "s"i? ":")? (LinkChar+) ) & {
-    let URL = [].concat.apply([], url).join('')
-    if (URL.length < 4) { // 4 chars is the shortest a URL can be (i.e. t.co)
+    // All words basically go into this function so lets not make it slow
+    const maybeQuote = url[0] || []
+    const maybeProtocol = url[1] || []
+    const maybeLink = [] = url[2] || []
```

Comment:
intended to be `const maybeLink = url[2] || []`?

Reply:
fixed, tnx